include/fs/fs.h

```diff
@@ -442,66 +447,66 @@ int fs_closedir(struct fs_dir_t *zdp);
  *
  * @param mp Pointer to the fs_mount_t structure
  *
- * @retval 0 Success
- * @retval -ERRNO errno code if error
+ * @retval 0 on success;
+ * @retval <0 negative errno code on error.
  */
 int fs_mount(struct fs_mount_t *mp);
 
 /**
  * @brief Unmount filesystem
  *
- * Perform steps needed for unmounting a file system like
+ * Perform steps needed to unmount a file system like
  * calling the file system specific unmount function and removing
  * the mount point from mounted file system list.
  *
- *
  * @param mp Pointer to the fs_mount_t structure
  *
- * @retval 0 Success
- * @retval -ERRNO errno code if error
+ * @retval 0 on success;
+ * @retval <0 negative errno code on error.
  */
 int fs_unmount(struct fs_mount_t *mp);
 
 /**
- * @brief Mount point read entry
+ * @brief Get path of mount point at index
  *
- * Read mount point entry
+ * Function iterates through list of mount points and returns directory name
+ * of mount point at given index.
+ * On success @p index is incremented and @p name is set to mount directory
+ * name.  If mount point with given @p index does not exist, @p name will be
+ * assigned @c NULL value
  *
```

Comment:
"~F~*This f*unction iterates through *the* list of mount points and returns *the* directory name of *the* mount point at *the* given index. On success @<>p index is incremented and @<>p name is set to *the* mount directory name.  If *a* mount point with *the* given @<>p index does not exist, @<>p name will be ~assigned~ *set to* @<>c NULL~ value~."

Yes, English is verbose.